crates/store/re_protos/proto/rerun/v0/log_msg.proto

```diff
@@ -0,0 +1,178 @@
+syntax = "proto3";
+
+package rerun.log_msg.v0;
+
+import "rerun/v0/common.proto";
+
+// Corresponds to `LogMsg::SetStoreInfo`. Used to identify a recording.
+message SetStoreInfo {
+    // A time-based UID that is used to determine how a `StoreInfo` fits in the global ordering of events.
+    rerun.common.v0.Tuid row_id = 1;
+
+    // The new store info.
+    StoreInfo info = 2;
+}
+
+// The type of compression used on the payload.
+enum Compression {
+    // No compression.
+    NONE = 0;
+
+    // LZ4 block compression.
+    LZ4 = 1;
+}
+
+// The encoding of the message payload.
+enum Encoding {
+    // We don't know what encoding the payload is in.
+    UNKNOWN = 0;
```

Comment:
are we adding this just because grpc suggests it's best practice to have 0 enum member? I've seen this used and unused in few big open source projects.  I just wonder if it will create unnecessary overhead and if we should just keep the one option we have, which is arrow ipc.

Reply:
Yes, that's one of the reasons why I added this. We also have some enums on the Rust side that already have an `unknown` variant (such as `StoreSource`), so I felt like it'd make sense for consistency.

Reply:
ok, cool, if there's any other enum variant in our spec that doesn't have 0 field, we should add it (follow up, ofc)